packages/psss/src/api/queries/useWeeklyReportForAlert.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import groupBy from 'lodash.groupby';
+import keyBy from 'lodash.keyby';
+import { reduceToDictionary } from '@tupaia/utils';
+import { getPeriodByDate } from '../../utils';
+import { combineQueries, useData, useReport } from './helpers';
+
+export const useWeeklyReportForAlert = alert => {
+  const { period, organisationUnit: countryCode, syndrome } = alert;
+
+  const reportParams = {
+    startWeek: period,
+    endWeek: getPeriodByDate(new Date()),
+    syndrome,
+  };
+  const query = combineQueries({
+    countryReport: useReport(`confirmedWeeklyReport/${countryCode}`, {
+      params: reportParams,
+    }),
+    siteReport: useReport(`weeklyReport/${countryCode}/sites`, {
+      params: reportParams,
+    }),
+    sites: useData(`country/${countryCode}/sites`),
```

Comment:
The combine queries is awesome. This is so nice to read and easy to understand.

Reply:
Thanks Tom! A bit inspired by GraphQL, dare I say 😃